### PR TITLE
Add step in VNet setup

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3676

### CODE REVIEW

File: astro/connect-azure.md [hunk 1]
```diff
@@ -68,13 +68,42 @@ This connection option is only available for dedicated Astro Hosted clusters and
...
+
+3. Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) to tell them that you have granted permissions to the Astronomer Service Principal. In addition, provide the following details in your request:
+
+    - Astro cluster ID and name
+    - Azure TenantID and Subscription ID of the target VNet
+    - Region of the target VNet
+    - VNet ID of the target VNet
+    - Classless Inter-Domain Routing (CIDR) block of the target VNet
 
 After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VNet. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VNet needs to accept the peering request and create the routing table entries in the target VNet.
```
Comment (jwitz) [inline astro/connect-azure.md:108]: How/ when does Astronomer delete the permissions that were just created? @afastronomer

Comment (afastronomer) [reply astro/connect-azure.md:108]: Once VNet peering connectivity is established and confirmed by CRE, customer can delete those role assignments.

Comment (afastronomer) [inline astro/connect-azure.md:108]: This part is applicable for AWS but Azure. It has to be something like:
```
After receiving your request, Astronomer support creates a VNet peering connectivity from both sides. No other actions are needed from your side. Astronomer support will let you know when connectivity is ready for use.
```

Comment (jwitz) [inline astro/connect-azure.md:108]:
```suggestion
After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VNet. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VNet needs to accept the peering request and create the routing table entries in the target VNet.

When the network connection is confirmed, you can delete the temporary roles you created using the [Azure CLI](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-remove#azure-cli).
```

File: astro/connect-azure.md [hunk 2]
```diff
@@ -68,15 +68,46 @@ This connection option is only available for dedicated Astro Hosted clusters and
...
+
+3. Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) to tell them that you have granted permissions to the Astronomer Service Principal. In addition, provide the following details in your request:
+
+    - Astro cluster ID and name
+    - Azure TenantID and Subscription ID of the target VNet
+    - Region of the target VNet
+    - VNet ID of the target VNet
+    - Classless Inter-Domain Routing (CIDR) block of the target VNet
+
+After receiving your request, Astronomer support creates a VNet peering connection from both sides. No other actions are needed from your side. Astronomer support will let you know when the connection is ready to use.
```
Comment (jwitz) [inline astro/connect-azure.md:108]:
```suggestion
After receiving your request, Astronomer support creates a VNet peering connection between the two VNets. No other actions are required from you. Astronomer support will notify you when the connection is ready to use.
```

**Review by lzdanski — APPROVED** [review]

File: astro/connect-azure.md [hunk 3]
```diff
@@ -68,15 +68,46 @@ This connection option is only available for dedicated Astro Hosted clusters and
...
+      --scope <resource-group-id>
+    
+    az role assignment create \
+      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
+      --role "Network Contributor" \
+      --scope <subscription-id>
+    ```
+
+3. Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) to tell them that you have granted permissions to the Astronomer Service Principal. In addition, provide the following details in your request:
+
```
Comment (afastronomer) [inline astro/connect-azure.md:101]: We do not need cluster name, ID is the most suitable information.
We do not need region and CIDR block.

File: astro/connect-azure.md [hunk 4]
```diff
@@ -68,15 +68,46 @@ This connection option is only available for dedicated Astro Hosted clusters and
...
+    - Astro cluster ID and name
+    - Azure TenantID and Subscription ID of the target VNet
+    - Region of the target VNet
+    - VNet ID of the target VNet
+    - Classless Inter-Domain Routing (CIDR) block of the target VNet
```
Comment (lzdanski) [inline astro/connect-azure.md:106]:
```suggestion
    - Astro cluster ID 
    - Azure TenantID and Subscription ID of the target VNet
    - VNet ID of the target VNet
```

File: astro/connect-azure.md [hunk 5]
```diff
@@ -68,15 +68,44 @@ This connection option is only available for dedicated Astro Hosted clusters and
...
+      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
+      --role "Network Contributor" \
+      --scope <subscription-id>
+    ```
+
+3. Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) to tell them that you have granted permissions to the Astronomer Service Principal. In addition, provide the following details in your request:
+
+    - Astro cluster ID 
+    - Azure TenantID and Subscription ID of the target VNet
+    - VNet ID of the target VNet
```
Comment (lzdanski) [inline astro/connect-azure.md:104]:
```suggestion
    - Astro Cluster ID 
    - Azure Tenant ID and Subscription ID of the target VNet
    - VNet ID of the target VNet
```

**Review by afastronomer — APPROVED** [review]

**Pull request merged**